### PR TITLE
Twitter Link changed to X

### PR DESCRIPTION
**Pull request by ansuman-shukla**
## Purpose / Description
_Link was outdated and needed to change_

## Fixes
* Isuue found myself not raised yet#

## Approach
_Went to README.md and updated the link from https://twitter.com/ankidroid to https://x.com/ankidroid_

Screenshot 2024-08-06 045024

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration (SDK version(s), emulator or physical, etc)

## Learning (optional, can help others)
_Describe the research stage_

_Links to blog posts, patterns, libraries or addons used to solve this problem_

## Checklist
_Please, go through these checks before submitting the PR._

- [ ] You have a descriptive commit message with a short title (first line, max 50 chars).
- [ ] You have commented your code, particularly in hard-to-understand areas
- [ ] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ x] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ Join Us On
 <a href="https://discord.gg/qjzcRTx"><img src="docs/graphics/logos/discord_logo_color.png" height="50px"/></a>
 <a href="https://www.reddit.com/r/Anki"><img src="docs/graphics/logos/reddit_logo_color.png" height="50px"/></a>
 <a href="https://www.facebook.com/AnkiDroid/"><img src="docs/graphics/logos/facebook_logo_color.png" height="50px"/></a>
-<a href="https://twitter.com/ankidroid"><img src="docs/graphics/logos/twitter_logo_color.png" height="50px"/></a>
+<a href="https://x.com/ankidroid"><img src="docs/graphics/logos/twitter_logo_color.png" height="50px"/></a>
```
Comment (Arthur-Milchior) [inline README.md:77]: Fine I guess
As in #16831 I'd like to ask you to also update the logo if you could, to be consistent

**Review by Arthur-Milchior — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ansuman-shukla) [reply README.md:77]: Sure @Arthur-Milchior , I can do it

**Comment by mikehardy** [comment]
Should go in as a pair with whatever the resolution of #16418 is

**Comment by mikehardy** [comment]
this will certainly conflict with #16837 when that goes through (it is in the queue now) so preemptively updated the PR here to match the new icon name going in there, should work out?

**Review by mikehardy — APPROVED** [review]

**Pull request merged**